cylc/uiserver/resolvers.py

```diff
@@ -316,11 +316,15 @@ async def play(cls, workflows, args, workflows_mgr, log):
 
                 if ret:
                     # command failed
-                    _, err = proc.communicate()
+                    out, err = proc.communicate()
+                    msg = out.strip() or err.strip() or (
```

Comment:
Might not want/need to include stdout in this.

https://github.com/cylc/cylc-flow/pull/5535/files#r1193518146

Reply:
Good idea!